### PR TITLE
move 'cursor: pointer' from tabBar to tabBar-content

### PR DESCRIPTION
**Pull request by kpge**
Signed-off-by: kpge <gekangping@126.com>
move 'cursor: pointer' from tabBar to tabBar-content 
* fix:#5538

cursor fixed

### CODE REVIEW

File: packages/core/src/browser/style/tabs.css [hunk 1]
```diff
@@ -47,6 +46,7 @@
   flex: none;
   align-items: flex-end;
   z-index: 1;
+  cursor: pointer;
```
Comment (magopian) [inline packages/core/src/browser/style/tabs.css]: How would that behave if the `p-TabBar` is in another `data-orientation`? (Does that ever happen?)

Comment (kpge) [reply packages/core/src/browser/style/tabs.css]: Thank you, I think you are right, it should be `pointer` no matter what `data-orientation` is.
I will modify it soon.

Comment (magopian) [reply packages/core/src/browser/style/tabs.css]: For reference, that's how I did it: https://github.com/theia-ide/theia/pull/5541/commits/f11ebc248afc4fd6de7cf0b8e5232d6f2ee2872b

Comment (kpge) [reply packages/core/src/browser/style/tabs.css]: Thank you a lot.

Comment (vince-fugnitto) [reply packages/core/src/browser/style/tabs.css]: I'll re-review :)

**Pull request merged**